contrib/test_jenkins.sh

```diff
@@ -659,6 +659,23 @@ check_make_distcheck() {
 	fi
 }
 
+check_config_h() {
+	echo 1..1 > check_config_h.tap
+
+	srcdir=$PWD/../src
+
+	# Check if all .c files include config.h
+	echo "==== Checking for config.h files in directory $srcdir ===="
+
+	if [ `find $srcdir -name \*.c |xargs grep -Le '#\s*include\s\+"config.h"'|wc -l` -eq 0 ];
+	then
+		echo echo "ok 1 - build successful " >> check_config_h.tap
+	else
+		echo Missing include config.h in files: `find $srcdir -name \*.c |xargs grep -Le '#\s*include\s\+"config.h"'`;
```

Comment:
can we cache the results of `find $srcdir -name \*.c |xargs grep -Le '#\s*include\s\+"config.h"'` command to the variable?
and then use it for the `if` statement and `echo`

Reply:
yep, why not... and use wc -w (word count) instead of -l (line count)